tools/link.py

```diff
@@ -1405,6 +1405,8 @@ def limit_incoming_module_api():
 
   if settings.WASM_WORKERS:
     settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$_wasmWorkerInitializeRuntime']
+    if settings.EMBIND:
+      settings.REQUIRED_EXPORTS.append('_embind_initialize_bindings')
```

Comment:
These two lines are duplicated in setup_pthreads.

How about moving them to the `if EMBIND` section on line 1383?